less/navigationBar.less

```diff
@@ -643,13 +643,17 @@
   .bookmarkButton {
     background: url('../img/toolbar/bookmark_btn.svg') center no-repeat;
     -webkit-mask-repeat: no-repeat;
-    width: 14px;
-    height: 14px;
-    margin-bottom: 2px;
-    margin-left: 6px;
+    background-size: 15px 15px;
+    margin: 0;
+    height: 0;
+    width: 15px;
+    height: 15px;
+    position: relative;
+    left: 5px;
```

Comment:
This is a magic number. At first I tried display: flex, justify-content: center, but it did not work well on macOS while it did on Ubuntu.